kubernetes/configurer/utils/fpl-process-definition.sh

```diff
@@ -2,22 +2,20 @@
 
 set -eu
 
-dir=$(dirname ${0})
-root_dir=$(realpath ${dir}/../../..)
+definition_dir=$1
+output_dir=$2
+definition_output_file_name=$3
```

Comment:
Having to pass 3 arguments is a bit complicated. Would be better IMO to accept just two, input path pointing to directory with JSON definitions and output XLS file.

Seeing `${dir}/utils/fpl-process-definition.sh ${root_dir}/ccd-definition ${build_dir}/ccd-definition.xlsx` is easier to reason about then `${dir}/utils/fpl-process-definition.sh ${root_dir}/ccd-definition ${build_dir} ccd-definition.xlsx`

Reply:
I think this is fine for now